### PR TITLE
Adding space routes to create, see all (+tests)  and connecting to ap…

### PR DESCRIPTION
**Pull request by Alexia-May**
We added:
- routes to create a space
- associated test
- we add to create a get all route (+ test) to confirm that the space was added

### CODE REVIEW

**Review by MichaelVerdon — APPROVED** [review]
Nice one guys!

File: database_recipe.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ CREATE TABLE users (
 -- Then the table with the foreign key second.
 CREATE TABLE albums (
   id SERIAL PRIMARY KEY,
-  title text,
+  title text,git
```
Comment (neilstudd) [inline database_recipe.md:134]: 👀 a wild `git` appears!

Comment (MyGollyGosh) [reply database_recipe.md:134]: What is this for please?

**Review by UnisM1 — APPROVED** [review]
Everything looks good!

File: tests/test_space_routes.py [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
+    ])
 
-#     assert response.status_code == 200
-#     assert response.data.decode("utf-8") == "Book added successfully"
+"""
+Test to see if we can list all spaces
+GET /all-space
+"""
+
+def test_get_spaces(db_connection, web_client):
```
Comment (MyGollyGosh) [inline tests/test_space_routes.py:36]: Did we want to add some jinga to this to start to make the UI of the spaces? Or are we happy just returning the string?

**Review by aulus-plautius — APPROVED** [review]

**Review by MyGollyGosh — APPROVED** [review]
Perfect

**Pull request merged**